client/app/User.class.js

```diff
@@ -0,0 +1,9 @@
+export default class {
+  constructor(user) {
+    Object.assign(this, user);
+  }
+
+  hasRecentAutorenew() {
+    return ['FR', 'CA', 'QC', 'WE', 'WS', 'ASIA', 'SG', 'AU'].includes(this.nichandle);
```

Comment:
Why not having these in a constant file ?

Reply:
I found it redundant as the aim is to go through this class to check this property but it could make sense for maintanability purpose